packages/ui/src/components/pagination/pagination-service.ts

```diff
@@ -0,0 +1,57 @@
+import { computed, ref } from 'vue';
```

Comment:
Why its called pagination-service with kebab notation? 
We can use composable notation with camelCase here.